docs/upgrade-notes/data-migrations-rds.md

```diff
@@ -0,0 +1,93 @@
+---
+id: data_migrations_rds
+title: Running data migrations for RDS
+hide_title: false
+---
+
+## Background
+
+This release of Cumulus (x.x.x) integrates with RDS and creates a new PostgreSQL database for archiving Cumulus data (e.g. granules, files, executions).
+
+While eventually Cumulus will only support using RDS as its data archive, for now the system will perform **parallel writes** to both DynamoDB and PostgreSQL so that all new data is archived in both datastores.
```

Comment:
```suggestion
While eventually Cumulus will only support using a PostgreSQL-compatible database as its data archive, for now the system will perform **parallel writes** to both DynamoDB and PostgreSQL so that all new data is archived in both datastores.
```